Wobble/Graphics/UI/BackgroundImage.cs

```diff
@@ -91,6 +88,26 @@ public override void Destroy()
             base.Destroy();
         }
 
+        /// <summary>
+        ///     Scale the background sprite to the virtual window.
+        /// </summary>
+        private void AutoResize() {
```

Comment:
Not really that big of a deal, but can you format the bracket onto a new line.

Reply:
Alright. I found a case that doesn't completely work with this code as-is (+100 on both dimensions)

Not just that, but apparently the pivot is off-center.